app/src/pages/CounterPage/containers/CounterPageContainer.js

```diff
@@ -0,0 +1,75 @@
+import React, {Component} from 'react';
+import Counter from '../components/Counter';
+import PropTypes from 'prop-types';
+
+class CounterPageContainer extends Component {
+    constructor(props) {
+
+        super(props);
+        this.state ={
+            countValue:0,
+            parity:'even',
+            countScreen:'countScreenR'
```

Comment:
what is countScreen? the name is not clear enough. 
As I understood correctly, it is a className. If it is, the best name for it will be `countScreenClassName`